docs/causes.md

```diff
@@ -133,10 +138,12 @@ Bing is known to run thousands of experiments a year, so they must have a standa
+Consider the following hypothetical example. An experimenter is testing an impact of their machine learning model for recommending car features on a car manufacturers website for potential buyers. The recommendations are shown in a search box. The control experience is a hardcoded list of car features which instantly returns a random set of five car features. The treatment is a machine learning model which takes a few hundred milliseconds to predict the five best features for the current user. Since not all users of the web page use the search feature, the experimenters would zoom-in only to those that interacted with the search box. Here is where defining the trigger condition will have a large impact on a potential SRM. For example, if the triggered condition is to keep all the users in the analysis that have seen the search box (e.g. have visited the site with this box) or have entered the search box (e.g. have clicked in the field), both treatment and control will have equal chances of triggering. That's fine. However, if the triggered condition is slightly more restrictive, e.g. all users that entered the search field and performed an action that retrieved recommendations, the groups would no longer be comparable. The time added in order to return the results from the machine learning model can be large enough for enough users to close the page/navigate away, resulting in their activity not being properly logged. As a result, less users will appear in the treatment group. Broader triggered condition is a a solution for most SRMs of this type.
```

Comment:
I think we could make this point more clearly with a simpler experiment:

```
Consider the following hypothetical example. A search engine is testing the impact of a complex, new search algorithm. The control search results load in ~30ms but the treatment search results take ~300ms to load. The experimenter decides to track assignment when the search results for each group have loaded. When the test is launched, not as many users in the treatment group waited for the new search results to load as they did in the control group. Since user assignment was only tracked after the search results loaded, the control group recorded more users than the treatment group.
```

Reply:
Cool. Thanks for making it more concise!